### ISSUE TITLE
Asset loading for development protocol reported broken on two fresh MacOS installations

### ISSUE THREAD

**Issue by jthrilly** [codaco/Network-Canvas#402]
Two users have reported that the development protocol assets do not load for them.

We will likely need to implement a keyboard shortcut to allow us to get meaningful debug info to help diagnose this issue. Creating a separate issue for that now.

**Comment by jthrilly** [comment]
Investigation of this might depend on #403 being implemented.

**Comment by pfsj** [comment]
This only occurs for me only on Mac for the developmental protocol (other protocols load the assets). It occurs on both the welcome screen and the sociogram. 

<img width="2048" alt="screen shot 2018-02-27 at 9 21 14 am" src="https://user-images.githubusercontent.com/10948594/36737295-5847d530-1ba0-11e8-9767-1ceb0b63f74c.png">
<img width="2048" alt="screen shot 2018-02-27 at 9 26 39 am" src="https://user-images.githubusercontent.com/10948594/36737291-55c536fe-1ba0-11e8-891d-d7010a2ecb0f.png">

**Comment by bryfox** [comment]
I see the same thing with the alpha-2 release build.

Inspecting the package contents, the app files are present (not packaged in an asar). I saw a couple of version differences between the release build and my local output from `pack`, and tried to reproduce a build with the same symptoms. I 'succeeded' a couple of times, but haven't been able to do it reliably; there's either something non-deterministic in that process or I missed whatever I was doing differently.

How was the existing release build produced? CI server or development machine? What are the exact steps? Is it reproducible with a fresh checkout?

**Comment by jthrilly** [comment]
@bryfox, it was produced using `electron-builder` which there is an npm task for (it should be a one-stop process), and just on my machine. I've tried the release build and I can't reproduce this issue, so this is really bizarre. 

To build yourself, I think you might need our signing certificates, which I will export for you when you get online later today.

**Comment by bryfox** [comment]
I'm building with my developer cert, which should be fine. I've been relying on `npm run pack`, because the dist tasks hangs when generating the .dmg (on that version of builder). I assume there's no difference there that would affect this. I remember you saying you were on an older version of Node recently, but haven't been able to consistently reproduce a 'bad' build with that either.

Note I see the release problem on another mac as well (confirmed on both Sierra & High Sierra).

**Comment by bryfox** [comment]
This was fixed in the previous release with some combination of a version update to electron-builder and file permissions.
